src/main/java/org/apache/sling/scriptingbundle/plugin/processor/Constants.java

```diff
@@ -53,6 +53,10 @@ private Constants() {}
     public static final String BND_SCRIPT_ENGINE_MAPPINGS = "scriptEngineMappings";
     public static final String BND_SEARCH_PATHS = "searchPaths";
 
+    public static final String VAULT_CONTEXT_XML = ".content.xml";
+    public static final String SLING_RESOURCE_SUPER_TYPE = "sling:resourceSuperType";
```

Comment:
There is a constant in Sling API

Reply:
I couldn't find a constant in Sling API for `sling:resourceSuperType`. In the whole Sling project I can only find 3 constants with this value, but none of them is in Sling API.

Reply:
https://sling.apache.org/apidocs/sling11/org/apache/sling/jcr/resource/api/JcrResourceConstants.html#SLING_RESOURCE_SUPER_TYPE_PROPERTY and dependency included with scope "provided"?